newIDE/app/src/ProjectManager/index.js

```diff
@@ -192,41 +479,91 @@ const ProjectManager = React.memo(
+      setProjectPropertiesDialogInitialTab('properties');
+    }, []);
+    const openProjectLoadingScreen = React.useCallback(() => {
+      setProjectPropertiesDialogOpen(true);
+      setProjectPropertiesDialogInitialTab('loading-screen');
+    }, []);
+    const onProjectPropertiesApplied = React.useCallback(
+      (options: { newName?: string }) => {
+        if (unsavedChanges) {
+          unsavedChanges.triggerUnsavedChanges();
+        }
+
+        if (options.newName) {
+          onChangeProjectName(options.newName);
+        }
+        setProjectPropertiesDialogOpen(false);
+      },
+      [unsavedChanges, onChangeProjectName]
+    );
+
+    const [openGameDetails, setOpenGameDetails] = React.useState<boolean>(
+      false
+    );
+    const projectUuid = project.getProjectUuid();
+    const { games, fetchGames } = useGamesList();
+    const { profile } = React.useContext(AuthenticatedUserContext);
+    const userId = profile ? profile.id : null;
+    React.useEffect(
+      () => {
+        fetchGames();
+      },
+      [fetchGames, userId]
+    );
+    const gameMatchingProjectUuid = games
+      ? games.find(game => game.id === projectUuid)
+      : null;
+    const onOpenGamesDashboardDialog = React.useCallback(
+      () => setOpenGameDetails(true),
+      []
+    );
+
     const [
       projectVariablesEditorOpen,
       setProjectVariablesEditorOpen,
     ] = React.useState(false);
+    const openProjectVariables = React.useCallback(() => {
+      setProjectVariablesEditorOpen(true);
+    }, []);
+
+    const [editedPropertiesLayout, setEditedPropertiesLayout] = React.useState(
+      null
+    );
+    const [editedVariablesLayout, setEditedVariablesLayout] = React.useState(
+      null
+    );
+    const onOpenLayoutProperties = React.useCallback((layout: ?gdLayout) => {
+      setEditedPropertiesLayout(layout);
+    }, []);
+    const onOpenLayoutVariables = React.useCallback((layout: ?gdLayout) => {
+      setEditedVariablesLayout(layout);
+    }, []);
+
     const [
       extensionsSearchDialogOpen,
       setExtensionsSearchDialogOpen,
     ] = React.useState(false);
+    const openSearchExtensionDialog = React.useCallback(() => {
+      setExtensionsSearchDialogOpen(true);
+    }, []);
     const [
       openedExtensionShortHeader,
       setOpenedExtensionShortHeader,
     ] = React.useState(null);
     const [openedExtensionName, setOpenedExtensionName] = React.useState(null);
-    const [openGameDetails, setOpenGameDetails] = React.useState<boolean>(
-      false
-    );
-
-    const projectUuid = project.getProjectUuid();
-    const gameMatchingProjectUuid = games
-      ? games.find(game => game.id === projectUuid)
-      : null;
 
+    const searchBarRef = React.useRef<?SearchBarInterface>(null);
+    const shouldAutofocusInput = useShouldAutofocusInput();
     React.useEffect(() => {
       if (!freezeUpdate && shouldAutofocusInput && searchBarRef.current) {
         searchBarRef.current.focus();
```

Comment:
This does not seem to work when testing it. Maybe you shoul add a timeout, the time that the drawer opens?

Reply:
I don't know why but a change in freezeUpdate does not seem to trigger the effect here

Reply:
Indeed, I should have added a todo because I forgot about it.

Reply:
You might want to either change the name of the props because it doesn't actually freezes the component, or memoize the whole component with React.memo and use the props in it

Reply:
Arriving at the end of the file, I just saw that you changed the arePropsEqual callback and the component is already memoized so we have the culprit! (Plus I can see your comment 😅)

Reply:
I think it's your comment actually. It must comes from the object tree.

Reply:
I think I will leave this small bug since we plan to pin the project manager.